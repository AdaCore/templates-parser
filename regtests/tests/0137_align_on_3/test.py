from test_support import *

gprbuild('alo3')
run('alo3')
