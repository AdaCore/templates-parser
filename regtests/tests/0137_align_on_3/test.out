1:----------------------------
       a  : int;       b : float
       r1 : boolean; r2  : character

2:----------------------------
       azerty : int; u              : float
       r      : boolean; this_is_it : character

3:----------------------------
       A            : Natural          := 0;
       Is_That_True : constant boolean := False;
       Value        := 12.3;
       No_Init      : Natural;
       Age          : Positive         := 12;

4:----------------------------
       A            : Natural          := 0;
       Is_That_True : constant boolean := False;
       Value        := 12.3;
       No_Init      : Natural;
       Age          : Positive         := 12;
